backend/src/middleware/slugify/uniqueSlug.spec.js

```diff
@@ -19,3 +19,17 @@ describe('uniqueSlug', () => {
     expect(uniqueSlug(string, isUnique)).resolves.toEqual('anonymous')
   })
 })
+
+describe('Slug is transliterated correctly', () => {
+  it('Converts umlaut to two letter equivalent', () => {
+    const umlaut = 'ä';
+    const isUnique = jest.fn().mockResolvedValue(true)
+    expect(uniqueSlug(string, isUnique)).resolves.toEqual('ae');
+  })
+
+  it('Removes Spanish enya ', () => {
+    const enya = 'ñ';
+    const isUnique = jest.fn().mockResolvedValue(true)
+    expect(uniqueSlug(string, isUnique)).resolves.toEqual('n');
+  })
+})
```

Comment:
@Mogge I wrote the tests but I could not figure out how to run the single unit test for the backend. I am running everything in Docker.

Reply:
Something is strange with this PR:

Tirokk wants to merge 4 commits into `pr3676base` from `pr3676head`

Can you please check this.

Did you try something like `docker-compose exec backend test src/middleware/slugify//`